src/Statement.cpp

```diff
@@ -84,11 +84,25 @@
 #include "util.h"
 #include "StringUtils.h"
 #include "VariableSelector.h"
+#include "Attribute.h"
 
 using namespace std;
 const Statement* Statement::failed_stm;
 
+static bool label_attr_generate;
+AttributeGenerator label_attr_generator;
```

Comment:
Please do not use global variables, but use static class variables.

Reply:
Done. It is now static variable of class `StatementFilter`.

Reply:
Sorry, I can't see `label_attr_generator` being a static variable of the class.

Reply:
> Sorry, I can't see `label_attr_generator` being a static variable of the class.

My apologies. I misunderstood previous comment about doing static variable. 

Now, I have declared `label_attr_generator` as static. Does your above comment means `static object of the class?` Please correct me if I am wrong?

Reply:
> Now, I have declared `label_attr_generator` as static. Does your above comment means `static object of the class?` Please correct me if I am wrong?

 It means put both variables into the `class` and make them `static`.